_includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-portal-core-feature.md

```diff
@@ -139,10 +139,10 @@ console transfer:generate
 
 ### 3) Set up the configuration
 
-**src/Pyz/Zed/SecurityMerchantPortalGui/SecurityMerchantPortalGuiConfig.php**
-
 Optional: For security reasons, we recommend enabling the security blocker feature that will block recurring attempts of resetting a password by setting `MERCHANT_PORTAL_SECURITY_BLOCKER_ENABLED` to `true;`
```

Comment:
Is this whole step optional or only `MERCHANT_PORTAL_SECURITY_BLOCKER_ENABLED`?

Reply:
The whole step.